pkg/sharings/sharings.go

```diff
@@ -225,7 +226,8 @@ func ShareDoc(instance *instance.Instance, sharing *Sharing, recStatus *Recipien
 		docType := rule.Type
 		// Trigger the updates if the sharing is not one-shot
 		if sharing.SharingType != consts.OneShotSharing {
-			if err := AddTrigger(instance, rule, sharing.SharingID); err != nil {
+			err := AddTrigger(instance, rule, sharing.SharingID)
+			if err != nil {
```

Comment:
Cosmetics reasons: I'm a bit maniac and the line exceeded 80 characters… I can revert it though.